setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
```

Comment:
Do we need to support distutils? setuptools provides som really useful functions like `find_packages`.